execution_engine/src/core/engine_state/mod.rs

```diff
@@ -1265,7 +1235,7 @@ where
                 correlation_id,
                 &preprocessor,
                 &protocol_version,
-                &protocol_data,
+                *standard_payment_hash,
```

Comment:
See related comment in get_deploy_metadata

```suggestion
                &system_contract_registry,
```

Reply:
Applied suggestion